server/schedule/operator/operator.go

```diff
@@ -770,8 +795,9 @@ func CreateMergeRegionOperator(desc string, cluster Cluster, source *core.Region
 		IsPassive:  false,
 	})
 
-	op1 := NewOperator(desc, source.GetID(), source.GetRegionEpoch(), kinds|kind|OpMerge, steps...)
-	op2 := NewOperator(desc, target.GetID(), target.GetRegionEpoch(), kinds|kind|OpMerge, MergeRegion{
+	brief := fmt.Sprintf("merge region: stores %v to %v", u64Set(source.GetStoreIds()), u64Set(target.GetStoreIds()))
```

Comment:
include target region id in brief?

Reply:
I don't think brief should contains region id. It's a little detailed.

Reply:
em, IMO for the merge operator, target region id is the most important information. It could a good idea to remove store ids if the message is too long.

Reply:
Sounds reasonable, I did it for the split operator, too.